docs/api/src.md

```diff
@@ -0,0 +1,123 @@
+<!-- front-matter
+id: api-src
+title: src()
+hide_title: true
+sidebar_label: src()
+-->
+
+# src()
+
+Creates a stream for reading [Vinyl][vinyl-concepts] objects from the file system.
+
+**Note:** Any UTF-8 BOMs will be removed from UTF-8 files read by `src()`, unless disabled using the `removeBOM` option.
```

Comment:
Add a description for BOM?

Reply:
I don't think it's important to understand what a BOM is for the purposes of gulp, but I agree that this could use extra clarity. I changed it to "BOMs have no purpose in UTF-8 and will be removed from UTF-8 files read by `src()`, unless disabled using the `removeBOM` option."